backend/zoho/api/types.go

```diff
@@ -0,0 +1,118 @@
+package api
+
+import (
+	"strconv"
+	"time"
+)
+
+// Time represents represents date and time information for Zoho
+// Zoho uses milliseconds since unix epoch (Java currentTimeMillis)
+type Time time.Time
+
+// UnmarshalJSON turns JSON into a Time
+func (t *Time) UnmarshalJSON(data []byte) error {
+	millis, err := strconv.ParseInt(string(data), 10, 64)
+	if err != nil {
+		return err
+	}
+	*t = Time(time.Unix(0, millis*int64(time.Millisecond)))
```

Comment:
time.Millisecond = 1000000

So this will overflow at 2^63 / 1000000 = 9223372036854 milliseconds from the epoch which turns out to be 2262 so we are OK here for 200 years!

Reply:
That's plenty time.